vslib/src/sai_vs_hostintf.cpp

```diff
@@ -1122,6 +1122,13 @@ sai_status_t vs_create_hostif_tap_interface(
         return SAI_STATUS_FAILURE;
     }
 
+    if (attr_type->value.s32 == SAI_HOSTIF_TYPE_GENETLINK)
```

Comment:
can you add comment why we are skipping creating SAI_HOSTIF_TYPE_GENETLINK here?

Reply:
Done